### DISCUSSION TITLE
程序员的浪漫: 婚礼邀请函小程序 - Qinng

### DISCUSSION THREAD

**Discussion by giscus** [qingwave/qingwave.github.io#79] [Announcements]
# 程序员的浪漫: 婚礼邀请函小程序 - Qinng

婚礼将近，作为一个有能耐好折腾的程序员怎么能不趁机展示下，着手开发个婚礼邀请函微信小程序。 总体设计选用微信小程序，传播方便，相对公众号定制性也更强。原本打算Github找一个改改，无奈不是太繁杂、就是

https://qingwave.github.io/wedding-invitation/

**Comment by yangsoon** [comment]
您好，您这个项目有开源吗？比较喜欢你这个风格，能不能fork一下

**Reply by qingwave** [reply]
@yangsoon 感谢你的认可，由于这个项目包含隐私信息比较多，目前短时间内不打算开源。如果确定自己开发，有什么问题欢迎交流哈

**Reply by 0x-fans** [reply]
现在这个开源了吗？哥们

**Reply by qingwave** [reply]
暂时没有哈，抱歉~
